client/src/m_menu.cpp

```diff
@@ -1638,10 +1706,17 @@ static void M_PlayerTeamChanged (int choice)
 
 static void SendNewColor(int red, int green, int blue)
 {
-	char command[24];
+	char colorcommand[24];
+	char customcolorcommand[30];// Acts 19 quiz
+	int colorpreset = D_ColorPreset(cl_colorpreset.cstring());// Acts 19 quiz
 
-	sprintf(command, "cl_color \"%02x %02x %02x\"", red, green, blue);
-	AddCommandString(command);
+	sprintf(colorcommand, "cl_color \"%02x %02x %02x\"", red, green, blue);
+	AddCommandString(colorcommand);
+	if (colorpreset == COLOR_CUSTOM)// Acts 19 quiz
+	{
+		sprintf(customcolorcommand, "cl_customcolor \"%02x %02x %02x\"", red, green, blue);
```

Comment:
Ditto.

Reply:
I added the one sprintf command because that was how the then-color command was being sent to the console (I renamed command to colorcommand since there would be two commands, and added customcolorcommand). If I'm understanding the request correctly, you want me to change not only my addition, but also how the standard color command is currently being handled since I'm already monkeying with that. (that's what I went with, anyway)

Reply:
Uh oh, I got the same Mac error that electricbass got on #386. Did I foul something up, or is something else going on?

Reply:
Something else is going on, I got that on my fork just from syncing it.